src/sdg/core/models/validators.py

```diff
@@ -79,3 +79,13 @@ def validate_specific_catalog(catalog):
                 'Een catalogus kan alleen naar een catalogus linken als "is_referentie_catalogus" is ingeschakeld.'
             )
         )
+
+
+def validate_ip_adress(array):
+    import ipaddress
+
+    for ip in array:
+        try:
+            ipaddress.ip_address(ip)
+        except ValueError:
+            raise ValidationError(_(f"'{ip}' is niet een valide ip adress"))
```

Comment:
```suggestion
            raise ValidationError(_(f"'{ip}' is geen valide IP-adres"))
```

Ik dacht trouwens dat Django al een IP-adres validator had. Check dat even.